packages/template-retail-react-app/config/default.js

```diff
@@ -24,9 +24,12 @@ module.exports = {
         commerceAPI: {
             proxyPath: `/mobify/proxy/api`,
             parameters: {
-                clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836',
-                organizationId: 'f_ecom_zzrf_001',
-                shortCode: '8o7m175y',
+                // clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836',
+                // organizationId: 'f_ecom_zzrf_001',
+                // shortCode: '8o7m175y',
+                clientId: '4114f443-49d7-41ac-93be-d83b2f932a4f',
+                organizationId: 'f_ecom_bgmj_stg',
```

Comment:
Are we changing instance?

Reply:
No.. this is something that I mentioned int he description above that I'll be changing back once reviewed. Reason being is that the API changes are only available on that instance.